Source/WebCore/platform/graphics/GlyphPage.h

```diff
@@ -75,10 +75,10 @@ class GlyphPage : public RefCounted<GlyphPage> {
 
     static unsigned count() { return s_count; }
 
-    static const unsigned size = 16;
+    static const unsigned size = 1 << 8; // 256 wold be enough for fitting all Latin-1 chars.
```

Comment:
typo: "wold"

Reply:
I'm running some perf tests. This was supposed to be a draft until we get any results there. I'll change the patch until then.

Reply:
If you upload with `git-webkit pr --draft` I won't get cc'd until it's ready next time :)